tensorflow/lite/experimental/examples/unity/TensorFlowLitePlugin/Assets/TensorFlowLite/SDK/Scripts/Interpreter.cs

```diff
@@ -27,29 +27,60 @@ namespace TensorFlowLite
   /// </summary>
   public class Interpreter : IDisposable
   {
-    private const string TensorFlowLibrary = "tensorflowlite_c";
+    public struct Options: IEquatable<Options> {
+      /// <summary>
+      /// The number of CPU threads to use for the interpreter.
+      /// </summary>
+      public int threads;
+
+      public bool Equals(Options other) {
+        return threads == other.threads;
+      }
+    }
+
+    public struct TensorInfo {
+      public string name { get; internal set; }
+      public DataType type { get; internal set; }
+      public int[] dimensions { get; internal set; }
+      public QuantizationParams quantizationParams { get; internal set; }
+
+      public override string ToString() {
+        return string.Format("name: {0}, type: {1}, dimensions: {2}, quantizationParams: {3}",
+          name,
+          type,
+          "[" + string.Join(",", dimensions) + "]",
```

Comment:
This line is failing in our internal test builds, where string.join only takes (string, string[]), not (string, int[]), can you fix?

Reply:
e.g., string.Join(",", dimensions.Select(d => d.ToString()).ToArray());